vendor/assets/javascripts/es5-shim.js

```diff
@@ -0,0 +1,1572 @@
+/*!
```

Comment:
Shouldn't this take care of that already? https://github.com/ad2games/ui_components/pull/14/files#diff-1243e347fae3300c9eaf5a552945a771R10